patch (orchestrator): [docs] add point and click instructions to create a new agent (#48)
Azure-Samples/openai-end-to-end-basic: README.md

### 2. Deploy an agent in the Azure AI Agent service

To test this scenario, you'll be deploying an AI agent. The agent uses a GPT model combined with a Bing search for grounding data. Deploying an AI agent requires data plane access to Azure AI Foundry. In this architecture, you will interact with the Azure AI Foundry portal and its resources over internet.

1. Open the Azure portal to your subscription.

1. Navigate to the Azure AI Foundry project named **projchat** in your resource group and open the Azure AI Foundry portal by clicking the Go to **Azure AI Foundry portal** button.

   This will take you directly into the 'Chat project'. Alternatively, you can find all your AI Foundry accounts and projects by going to <https://ai.azure.com> and you do not need to use the Azure portal to access them.

1. Click **Agents** in the side navigation.

1. Click **+ New agent** button.

1. From the Setup panel, change the Agent name to 'Baseline Chatbot Agent'.

1. Click **+ Add** button in the Knowledge section.

1. From the popup window select the knowledge type 'Grounding with Bing Search'. 

> 1. Then choose the existing connection named 'bingaiagent' and click **Connect** button.

Reviewer: ```suggestion
1. Then choose the existing connection named 'bingaiagent' and click the **Connect** button.
```

1. Deploy the agent.

>    Here you'll deploy your orchestration agent by creating it directly from the Azure AI Foundry portal's.

Reviewer: ```suggestion
   Here you'll deploy your orchestration agent by creating it directly from your Azure AI Foundry portal.
```

Author: I've removed this line in favor of another fleshed out sentence we had above. Please let's revisit if this reads ok now

>    1. In your browser of preference, open the Azure portal to your subscription.

Reviewer: ```suggestion
   1. Open the Azure portal to your subscription.
```

Author: like it @ckittel 

done | addressed from e9a35b0b1904c6ea71264559ef88b684648a14a9

>    1. Sign in to the Azure portal

Reviewer: ```suggestion
```

Author: removed @ckittel, if this was not really desired just ask and I will revert

done | addressed from e23b1c86ac1227683284f7ea4ec0c7a5b7a9896b

> 

Author: removed @ckittel, if this was not really desired just ask and I will revert

done | addressed from e23b1c86ac1227683284f7ea4ec0c7a5b7a9896b

>    1. Navigate to the Azure AI Foundry project named **projchat** in your resource group and open the Azure AI Foundry portal by clicking the Go to Azure AI Foundry portal button.

Reviewer: ```suggestion
   1. Navigate to the Azure AI Foundry project named **projchat** in your resource group and open the Azure AI Foundry portal by clicking the **Go to Azure AI Foundry portal** button.
```

Author: good catch @ckittel 

done | addressed from fe956fb6664117f287286c450a21949d23d597d4

>       This will take you directly into the 'Chat project'. Alternatively, you can find all your AI Foundry accounts and projects by going to https://ai.azure.com and you do not need to use the Azure portal to access them.

Reviewer: ```suggestion
      This will take you directly into the 'Chat project'. Alternatively, you can find all your AI Foundry accounts and projects by going to <https://ai.azure.com> and you do not need to use the Azure portal to access them.
```

>    1. Click Agents in the side navigation.

Reviewer: ```suggestion
   1. Click **Agents** in the side navigation.
```

Author: good catch @ckittel 

done | addressed from fe956fb6664117f287286c450a21949d23d597d4

   1. Click **+ New agent** button.

>    1. From the **Setup** panel, change the Agent name to 'Baseline Chatbot Agent'.

Reviewer: ```suggestion
   1. From the Setup panel, change the Agent name to 'Baseline Chatbot Agent'.
```

Author: good catch @ckittel 

done | addressed from fe956fb6664117f287286c450a21949d23d597d4

>    1. Scroll down and click **+ Add** button in the Knowledge section.

Reviewer: ```suggestion
   1. Click **+ Add** button in the Knowledge section.
```

Author: good catch @ckittel 

done | addressed from fe956fb6664117f287286c450a21949d23d597d4

   1. From the popup window select the knowledge type 'Grounding with Bing Search'. 

   1. Then choose the existing connection named 'bingaiagent' and click **Connect** button.

> ### 3. Test the agent from the Azure AI Foundry portal in the playground. *Optional.*

Reviewer: ```suggestion
### 3. Test the agent from the Azure AI Foundry portal in the playground.
```

Author: good call @ckittel 

done | addressed from c21d714f14dbc0b987a6d043fb36c442809404e1

Here you'll test your orchestration agent by invoking it directly from the Azure AI Foundry portal's playground experience.

> *This step testing step is completely optional and assumes you followed steps from the Deploy the agent section above*

Reviewer: ```suggestion
```

Author: good call @ckittel 

done | addressed from c21d714f14dbc0b987a6d043fb36c442809404e1

> 

Reviewer: ```suggestion
```

Author: good call @ckittel 

done | addressed from c21d714f14dbc0b987a6d043fb36c442809404e1

Workloads build chat functionality into an application. Those interfaces usually call Azure AI Foundry project endpoint invoking a particular agent. This implementation comes with such an interface. You'll deploy it to Azure App Service using the Azure CLI.

1. Generate some variables to set context.

   *The following variables align with the defaults in this deployment. Update them if you customized anything.*

   ```bash
   AI_FOUNDRY_NAME="aif${BASE_NAME}"
   AI_FOUNDRY_PROJECT_NAME="projchat"
   AI_FOUNDRY_AGENT_CREATE_URL="https://${AI_FOUNDRY_NAME}.services.ai.azure.com/api/projects/${AI_FOUNDRY_PROJECT_NAME}/assistants?api-version=2025-05-15-preview"

   echo $AI_FOUNDRY_AGENT_CREATE_URL
   ```

> 1. Get Agent Id value.

Reviewer: ```suggestion
1. Get Agent ID value.
```

Author: great call out @ckittel 

done | addressed from aca4d1445b115633ad62a545421191f769dd9e20
